### PR TITLE
Adds API for fast-reconnect support in CC32xx.

### PR DESCRIPTION
**Pull request by balazsracz**
Also add a wlan_disconnect call.

### CODE REVIEW

File: src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx [hunk 1]
```diff
@@ -31,6 +31,8 @@
  * @date 18 March 2016
  */
 
+#define LOGLEVEL INFO
```
Comment (bakerstu) [inline src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx]: Disable logging

File: src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx [hunk 2]
```diff
@@ -113,6 +114,16 @@ public:
...
     struct FatalErrorEvent;
     
+    /** The Wlan reconnect policy */
+    enum WlanConnectionPolicy {
+        WLAN_CONNECTION_NO_CHANGE,
+        /// Scan for wifi networks and connect to the nearest that is stored in
+        /// the profiles (by profile priority and security settings).
+        WLAN_CONNECTION_SCAN,
+        /// Reconnect to the last connected AP.
+        WLAN_CONNECTION_RECONNECT
```
Comment (bakerstu) [inline src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx]: ??? WLAN_CONNECTIN_FAST_RECONNECT ???  Maybe...

File: src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx (hunk 1 above)
Comment (balazsracz) [reply src/freertos_drivers/net_cc32xx/CC32xxWiFi.cxx]: done

File: src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx (hunk 2 above)
Comment (balazsracz) [reply src/freertos_drivers/net_cc32xx/CC32xxWiFi.hxx]: done

**Pull request merged**